simapp/app.go

```diff
@@ -598,6 +598,7 @@ func NewSimApp(
 	app.SetBeginBlocker(app.BeginBlocker)
 	app.SetEndBlocker(app.EndBlocker)
 	app.setAnteHandler(txConfig)
+	app.SetPrecommiter(app.Precommiter)
```

Comment:
For consistency should do it as well in pre blocker here?

Reply:
whoops, missed that